cfme/infrastructure/config_management.py

```diff
@@ -520,3 +431,103 @@ def __init__(self, name=None, url=None, ssl=None, credentials=None, key=None):
         self.ssl = ssl
         self.credentials = credentials
         self.key = key or name
+
+
+@navigator.register(ConfigManager, 'All')
+class MgrAll(CFMENavigateStep):
```

Comment:
Agreed we can't have classes with the same name in the same module, so we need to rename them something